src/hooks/useTheme.ts

```diff
@@ -6,7 +6,6 @@ import { useAuth } from './useAuth';
 export const useTheme = () => {
   const { user } = useAuth();
   const { theme, setTheme } = useContext(ThemeContext);
-
```

Comment:
Why remove this?

Reply:
i think i did this accidentally, apologies.